### ISSUE TITLE
Any way we can get the logits?

### ISSUE THREAD

**Issue by panagiotidi** [SYSTRAN/faster-whisper#577]
Is there any way that we can invoke and get the direct output of the decoder, i.e., the logits?

**Comment by panagiotidi** [comment]
Ok. Just saw the CTranslate2 does not return logits for words yet ([issue](https://github.com/OpenNMT/CTranslate2/pull/1489))

**Comment by MahmoudAshraf97** [comment]
CT2 now supports returning logits
